GrowthBook.Tests/ApiTests/LoadFeaturesTests.cs

```diff
@@ -0,0 +1,32 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Microsoft.Extensions.Logging;
+using Xunit;
+
+namespace GrowthBook.Tests.ApiTests;
+
+public class LoadFeaturesTests
+{
+    private const string ApiKey = "<AN API KEY GOES HERE>";
+
+    [Fact(Skip = "This is here as a shortcut for the purpose of debugging this SDK against the actual API and should not be used otherwise")]
```

Comment:
I don't like including these things t.b.h

Reply:
Yeah, I was a little on the fence about that as well. Removed.